maintenance/monitor-component-metrics.md

```diff
@@ -3,43 +3,261 @@ title: Monitor Calico component metrics
 
-Using the open-source Prometheus monitoring and alerting toolkit, you can view time-series metrics from {{site.prodname}} components in the Prometheus or Grafana interfaces.  
+## Requirements
 
-### Features
+In this tutorial we assume that you have completed all other introductory sections and possess a running Kubernetes cluster with {{site.prodname}} and calicoctl installed.
 
-This how-to guide uses the following {{site.prodname}} features:
+> **Note** Typha implementation is optional, if you don't have Typha in your cluster you should skip step 4 and 7 of this tutorial.
+   {: .alert .alert-warning}
 
-**Felix** and **Typha** components configured with Prometheus configuration parameters (for consumption by Prometheus).
+# How to
 
-### Concepts
+In this section we will go through the necessary steps to implement basic monitoring with Prometheus. You will assign required permission to a service account, create a permanent configuration and store it via configmaps, create services to discover endpoints dynamically, create a Prometheus pod, change Felix configuration and create a basic graph using consumed Felix metrics.
 
-#### About Prometheus
+## 1. Allowing user access to metrics
 
-The Prometheus monitoring tool scrapes metrics from instrumented jobs and displays time series data in a visualizer (such as Grafana). For {{site.prodname}}, the “jobs” that Prometheus can harvest metrics from are the Felix and Typha components. 
+You need to give a user account required permissions to be able to collect information from the nodes.
 
-#### About {{site.prodname}} Felix and Typha components
+> **Note**: A comprehensive guide to user roles and authentication can be [found at this link](https://kubernetes.io/docs/reference/access-authn-authz/rbac/).
+   {: .alert .alert-info}
 
-**Felix** is a daemon that runs on every machine that provides endpoints ({{site.prodname}} nodes). Felix is the brains of {{site.prodname}}. Typha is an optional daemon that extends Felix to scale traffic between {{site.prodname}} nodes and the datastore. **Typha** is used to avoid bottlenecks and performance issues in the datastore when you have over 50 {{site.prodname}} nodes. 
+```bash
+kubectl apply -f - <<EOF
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  name: prometheus-rbac
+subjects:
+  - kind: ServiceAccount
+    name: default
+    namespace: default
```

Comment:
Can we put this in a `calico-monitoring` namespace or similar, instead of the default namespace?